packages/graphql-client/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@affine/graphql-client",
```

Comment:
why not just `@affine/graphql`, where we put all gql/schema/codegen stuffs in, and `GraphQLClient` that used to send gql request as well